pkg/controller/jenkins/configuration/base/reconcile.go

```diff
@@ -113,11 +117,40 @@ func (r *ReconcileJenkinsBaseConfiguration) Reconcile() (reconcile.Result, jenki
 		return reconcile.Result{Requeue: true}, nil, r.Configuration.RestartJenkinsMasterPod(restartReason)
 	}
 
+	currentJenkinsVersion := jenkinsClient.GetVersion()
+	r.logger.V(log.VDebug).Info(fmt.Sprintf("Jenkins version: %s", currentJenkinsVersion))
+
+	jenkinsOutdated, latestVersion, err := isJenkinsOutdated(jenkinsClient.GetVersion())
+	if err != nil {
+		r.logger.Info(fmt.Sprintf("Cannot check current Jenkins version: %s", err))
+	} else if jenkinsOutdated {
+		// TODO: Add notification, not just only logger ino
+		r.logger.Info(fmt.Sprintf("Jenkins is outdated! Current version: %s, latest version: %s", currentJenkinsVersion, latestVersion))
```

Comment:
Don't think we need a notification for this, having logs is enough really. 
Instead of checking if the version is outdated, can we check if a new version exists.
What I mean here is that the word "outdated" had different connotations, and a version which is slightly older needn't be outdated everytime.

Reply:
https://github.com/jenkinsci/kubernetes-operator/issues/226